core/drivers/gic.c

```diff
@@ -950,8 +952,13 @@ static void __maybe_unused gic_native_itr_handler(void)
 
 	if (id <= gd->max_it)
 		interrupt_call_handlers(&gd->chip, id);
+	else if (IS_ENABLED(CFG_ARM_GICV3) && id >= 1020 && id <= 1023)
```

Comment:
Please add a comment about these IDs.
I think it would be useful to have a DMSG() even for these IDs since we normally don't receive them and if it happens a lot it might be a sign that something is wrong in the system.